src/main/java/com/zendesk/maxwell/schema/Table.java

```diff
@@ -50,15 +50,16 @@ private void initColumnOffsetMap() {
 		int i = 0;
 
 		for(ColumnDef c : columnList) {
-			this.columnOffsetMap.put(c.getName(), i++);
+			this.columnOffsetMap.put(c.getName().toLowerCase(), i++);
```

Comment:
Is this to protect against existing entries? I see column defined with lowercase name always below.